iamaccessgroupsv2/iam_access_groups_v2_examples_test.go

```diff
@@ -58,6 +56,13 @@ var _ = Describe(`IamAccessGroupsV2 Examples Tests`, func() {
 		testProfileID       string
 		testClaimRuleID     string
 		testClaimRuleEtag   string
+		testPolicyTemplateID	string
+		testAccountGroupID		string
+		testTemplateId		string
+		testTemplateVersionEtag	string
+		testTemplateLatestVersionEtag	string
+		testAssignmentID	string
+		testAssignmentEtag	string
```

Comment:
Please format the code to follow the style above: the variable types on each line is in the same column.
`go fmt iamaccessgroupsv2/*.go`